### ISSUE TITLE
`move` keyword is eaten sometimes by rustfmt

### ISSUE THREAD

**Issue by oli-obk** [nrc/rustfmt#53]
the following code
```rust
fn main() {
    let _ = function(move || 5);
    let _ = move || 42;
}
```
is changed to
```rust
fn main() {
    let _ = function(|| 5);
    let _ = move || 42;
}
```

related: https://github.com/rust-lang/rust/issues/24986

**Comment by nrc** [comment]
Probably not easy, actually, probably needs to fix the bug in libsyntax first

**Comment by oli-obk** [comment]
This is now fixed in rust master. Needs a regression test.

**Comment by nrc** [comment]
Fixed in #70
